src/module_3/module_3.py

```diff
@@ -0,0 +1,129 @@
+# %% Import packages
+import pandas as pd
+
+from sklearn.model_selection import train_test_split
+from sklearn import linear_model
+from sklearn.linear_model import Ridge
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.ensemble import RandomForestClassifier
+from sklearn import metrics
+from sklearn.preprocessing import normalize
+
+import matplotlib.pyplot as plt
+
+# %% Load data
+address = "D:\\Yo\\"
+filename = "feature_frame.csv"
+df = pd.read_csv(address + filename)
+
+# %% Data preprocessing
+
+# Obtain the id's for the orders where 5 or more products where bought
+filter_df = df[df["outcome"]==1]
+filter_df = filter_df.groupby("order_id").filter(lambda x: len(x) > 4)
+
+# Filter the data so we retain only those ids
+dataset = df[df["order_id"].isin(filter_df["order_id"])]
+
+# Change date format to retain only year, month and day
+dataset["created_at"] = dataset["created_at"].str[:-9]
```

Comment:
I am not sure what the format of the original date is here, but normally is better to do first `pd.to_datetime(dataset["created_at"])` in order to cast the column to a date, and then you can just convert it to whatever string you want in a safe way